.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Refined Github Discussions
+    url: https://github.com/sindresorhus/refined-github/discussions
```

Comment:
This is more like "Issues that don't have a specific topic" or that just don't match the other 3. I don’t think we'll enable discussions here

Reply:
Ok, I made a separate 'Discussions' as a single textarea,